### ISSUE TITLE
app crashing on startup after the  7.7-beta1 update

### ISSUE THREAD

**Issue by plokm21** [KotatsuApp/Kotatsu#1178]
### Brief summary

the app started crashing on startup after updating to the new update
it's actually very similar to another bug here that happened months ago i had the same problem in that thread as well
https://github.com/KotatsuApp/Kotatsu/issues/1078

it even has the same workaround where i open it in offline mode and then turn on wifi as a workaround  like i described here
https://github.com/KotatsuApp/Kotatsu/issues/1078#issuecomment-2328589124

### Steps to reproduce

open the app and it crashes after a few seconds  

### Kotatsu version

 7.7-beta1 

### Android version

6.0.1

### Acknowledgements

- [X] This is not a duplicate of an existing issue. Please look through the list of [open issues](https://github.com/KotatsuApp/Kotatsu/issues) before creating a new one.
- [X] This is not an issue with a specific manga source. Otherwise, you have to open an issue in the [parsers repository](https://github.com/KotatsuApp/kotatsu-parsers/issues/new/choose).

**Comment by Koitharu** [comment]
I cannot reproduce it so need some more info. Can you please take a look on [this](https://github.com/TeamNewPipe/NewPipe/issues/10516#issuecomment-1784712344)?

**Comment by plokm21** [comment]
> I cannot reproduce it so need some more info. Can you please take a look on [this](https://github.com/TeamNewPipe/NewPipe/issues/10516#issuecomment-1784712344)?

If you're asking  to unblock the trackers i don't know how to do that i don't think my phone has that feature

**Comment by Koitharu** [comment]
@plokm21 just to confirm you didn't turn any components off

**Comment by plokm21** [comment]
> @plokm21 just to confirm you didn't turn any components off

I'm not really sure how to do this i can't find the option on my app manager 
but I'm sure i haven't touched my phone settings in months and the app worked fine before the update

**Comment by Koitharu** [comment]
Ok, got it

**Comment by Koitharu** [comment]
Can you please check if it is reproducible on [v7.7-rc1](https://github.com/KotatsuApp/Kotatsu/releases/tag/v7.7-rc1)?

**Comment by plokm21** [comment]
Sorry it's still crashing but at least the work round still works so the app is perfectly usable outside of the bug

**Comment by Koitharu** [comment]
Can anyone provide a full crash stacktrace?

**Comment by Ero-gamer** [comment]
> Can anyone provide a full crash stacktrace?

App crash without generating crash logs.

**Comment by PrivacyFriendlyMuffins** [comment]
Same thing is happening to me.

I launch the app, immediately getting an error message saying:

> ⚠️ An error occurred
>
> Something went wrong. Please submit a bug report to the developers to help us fix it.

A smaller error message flashes twice on the screen:

> A migration from 24 to 23 was required but not found. Please provide the necessary Migration Path via RoomDatabase.Builder.AddMigration(Migration...) or allow for destructive migrations via one of the Roo... [It cuts off there.]

The app stays open for about 2½ seconds and then crashes to desktop (err...home screen?). The first error message persists until I either press "Send" or tap outside it, while the smaller "migration"-related error message goes away with the CTD.

---

- **Install Method**: F-Droid
- **App Version**: 7.7.9
- **OS Version**: Android 11
- **Hardware**: Motorola One 5G Ace

**Comment by bugcatcherlion** [comment]
Same issue, a message appears saying

> A migration from 24 to 23 was required but not found. Please provide the necessary Migration Path via RoomDatabase.Builder.AddMigration(Migration...) or allow for destructive migrations via one of the RoomDatabase. Builder.fallbackToDestructiveMigration*methods.

I am also unable to create a backup, the same error message as before appears when I attempt to create a new backup.

**Install Method**: F-Droid
**App Version**: 7.7.9
**OS Version**: Android 11
**Hardware**: Oppo A16k

**Comment by AcidentalHeart** [comment]
Same issue as the last two.

Install Method: F-Droid
App Version: 7.7.9
OS Version: Android 14
Hardware: Samsung Galaxy A13

**Comment by navin09** [comment]
Same. 

> A migration from 24 to 23 was required but not found. Please provide the necessary Migration Path via RoomDatabase.Builder.AddMigration(Migration...) or allow for destructive migrations via one of the RoomDatabase. Builder.fallbackToDestructiveMigration*methods.

Install Method: **F-Droid**
App Version: **7.7.9**
OS Version: **Android 15**
Hardware: **OnePlus 11G CPH2447**

**Comment by Locorock** [comment]
Reposted this to https://github.com/KotatsuApp/Kotatsu/issues/1265 since it looks like a different issue from the one in the original post
